ziti/ziti.go

```diff
@@ -79,13 +79,16 @@ type Context interface {
 	// Close closes any connections open to edge routers
 	Close()
 
-	// Add a Ziti MFA handler, invoked during authentication
+	// AddZitiMfaHandler adds a Ziti MFA handler, invoked during authentication
 	AddZitiMfaHandler(handler func(query *edge.AuthQuery, resp func(code string) error) error)
 	EnrollZitiMfa() (*api.MfaEnrollment, error)
 	VerifyZitiMfa(code string) error
 	RemoveZitiMfa(code string) error
 
 	Reload() error
+
+	SetAuthToken(token string)
```

Comment:
In my opinion, we're at the point where we need to add doc along with any functionality we expect people to use/consume. Can this get doc added along with 'how to use it' as well, please?

Reply:
Got you loud and clear. Will address on the morrow.